pkg/cmd/pr/reopen/reopen.go

```diff
@@ -30,9 +30,9 @@ func NewCmdReopen(f *cmdutil.Factory, runF func(*ReopenOptions) error) *cobra.Co
 	}
 
 	cmd := &cobra.Command{
-		Use:   "reopen {<number> | <url> | <branch>}",
+		Use:   "reopen [<number> | <url> | <branch>]",
 		Short: "Reopen a pull request",
-		Args:  cobra.ExactArgs(1),
```

Comment:
Same as `close`: we wanted to force the explicit argument because of the destructiveness of `close/reopen`.